src/navigation/mobileStepper/MobileStepper.jsx

```diff
@@ -0,0 +1,184 @@
+            alignItems: 'center',
+            justifyContent: activeStep === 0 ? 'center' : 'space-between',
+            gridTemplateAreas:
+                activeStep === 0 ? `
+                    "nextButton"
+                    "stepperIndicator"
+            ` :
+                    `
+                "backButton callToAction nextButton"
+                "stepperIndicator stepperIndicator stepperIndicator"
+            `,
+        },
+        stepperIndicator: {
+            alignItems: 'center',
+            display: 'flex',
+            gridArea: 'stepperIndicator',
+            justifyContent: 'center',
+        },
+        step: {
+            background: palette.grey[300],
+            borderRadius: 99,
+            display: 'inline-block',
+            height: 8,
+            marginRight: 5,
+            width: 8,
+        },
+        stepActive: {
+            animation: '$fadeIn 250ms ease-in-out',
+            background: palette.active.primary,
+        },
+        backButton: {
+            background: palette.grey[400],
+        },
+        nextButton: {
+            gridArea: 'nextButton',
+        },
+        '@keyframes fadeIn': {
+            '0%': {
+                opacity: 0,
+                transform: 'scale(0)',
+            },
+            '50%': {
+                opacity: 1,
+                transform: 'scale(1.4)',
+            },
+            '100%': {
+                opacity: 1,
+                transform: 'scale(1)',
+            },
+        },
+    }));
+
+    const classes = useStyles();
+
+    const backButton =
+        activeStep > 0 && steps - 1 ? (
+            <Button
+                className={classes.backButton}
+                id={`${id}--button_back`}
+                onClick={backStepAction}
+                width={90}
+            >
+                {backStepLabel}
+            </Button>
+        ) : null;
+
+    const lastButton =
+        activeStep === steps - 1 ? (
+            <Button
+                className={classes.nextButton}
+                color="success"
+                id={`${id}--button_last`}
+                onClick={lastStepAction}
+                width={90}
+            >
+                {lastStepLabel}
+            </Button>
+        ) : null;
+
+    const nextButton =
+        activeStep < steps - 1 ? (
+            <Button
+                className={classes.nextButton}
+                color="primary"
+                id={`${id}--button_next`}
+                onClick={nextStepAction}
+                width={activeStep === 0 ? 200 : 90}
+            >
+                {nextStepLabel}
+            </Button>
+        ) : null;
+
+    const callToActionButton =
+        !isEmpty(callToAction) && activeStep > 0 ? (
+            <Button
+                className={classes.callToAction}
+                color="outline"
+                id={`${id}--button_call_to_action`}
+                onClick={() => window.open(callToAction.url)}
```

Comment:
I think we should leave the logic of UX to the parent component. We may not always want to open up a new window.

Reply:
In fact I am not sure that this call to action should even be in this component. Mobile Stepper should just be concerned with providing a dots indicator and back and next buttons.